server/app/controllers/dev/seeding/CategoryTranslationFileParser.java

```diff
@@ -74,7 +80,24 @@ private void parseCategoryTranslationFiles() {
       logger.error("File is not a directory: " + CATEGORY_TRANSLATIONS_DIRECTORY);
       return;
     }
-    File[] files = directory.listFiles();
+    File[] files = null;
+    try {
+      files = directory.listFiles();
+    } catch (SecurityException e) {
+      logger.error(
+          "There was a security exception listing files in directory: "
+              + CATEGORY_TRANSLATIONS_DIRECTORY,
+          e);
+    }
+
+    if (files == null) {
+      logger.error(
+          "Unable to list files in directory: "
+              + CATEGORY_TRANSLATIONS_DIRECTORY
+              + ".  "
+              + "There may have been an I/O error.");
+      return;
+    }
     for (File file : files) {
       String fileLanguage = FilenameUtils.getExtension(file.getName());
```

Comment:
(blocking, question) You're using the file extension to determine the language here, but the language file in this PR are all `txt` files. Are the files named as expected?

Reply:
OMG, I'm so glad you caught that.  Those `.txt` files didn't belong in there.  That was the result of me having worked on this branch a while ago and forgetting that I'd since changed the names of the files.

Reply:
Gotcha :)